lib/ansible/modules/cloud/google/gcp_appengine_firewall_rule_info.py

```diff
@@ -29,7 +29,7 @@
 
 DOCUMENTATION = '''
 ---
-module: gcp_appengine_firewall_rule_facts
+module: gcp_appengine_firewall_rule_info
 description:
 - Gather facts for GCP FirewallRule
```

Comment:
Missed a change here - this module still uses `facts` in the description and doesn't have the "was previously called" line.

Reply:
That's because the module was added after stable-2.8 was branched, so it will first be released in Ansible 2.9